src/watertap_contrib/seto/analysis/net_metering/PV_RO.py

```diff
@@ -0,0 +1,457 @@
+import os
```

Comment:
We really should add a test file for this. Something is better than nothing.

Reply:
I was thinking about this yesterday and agree. @zacharybinger can you provide a simple test file for the RO+PV flowsheet?